kong/conf_loader.lua

```diff
@@ -3,39 +3,45 @@ local DEFAULT_PATHS = {
   "/etc/kong/kong.conf"
 }
 
-local CONF_SCHEMA = {
-  -- kong
-  log_level = {enum = {"debug", "info", "notice", "warn",
-                       "error", "crit", "alert", "emerg"}},
-
-  ssl = {typ = "boolean"},
-
-  custom_plugins = {typ = "array"},
-
+local CONF_PARTIAL_SCHEMA = {
```

Comment:
`CONF_INFERENCES`? Since, as we discussed but for public record, this table has no authority over the value allowed in the config file, but only serves to enforce the type of said values.